twisted/web/_http2.py

```diff
@@ -625,7 +625,8 @@ def _handleWindowUpdate(self, event):
                 # unnecessary but benign. We'll ignore it.
                 return
 
-            self.priority.unblock(streamID)
+            if self._outboundStreamQueues.get(streamID):
```

Comment:
minor comment

rather than this `if: do` , maybe we can go with `try: do: catch:`

```
try:
   self.priority.unblock(streamID)
except IndexError:
  pass
```

I don't know if catching this generic IndexError is a good idea ... maybe prority.unblock() can be changed to raise a more specific error 

again, just a minor comment